### PR TITLE
Doc201 optimisation display

### PR DESCRIPTION
**Pull request by athoynilimanew**
## Linked Issues

#3504

## Checklist

I confirm that I have completed the following checks:

- [x] Tests run locally and pass `pytest tests --reactor`
- [x] Code quality checks run locally and pass `pre-commit run --from-ref develop --to-ref HEAD`
- [x] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

File: bluemira/optimisation/_nlopt/conditions.py [hunk 1]
```diff
@@ -51,6 +56,14 @@ def _validate(self) -> None:
             self.max_eval = int(self.max_eval)
 
     def _no_stopping_condition_set(self) -> bool:
+        """
+        Checks if all stopping conditions are unset.
+
+        Returns
+        -------
+        bool :
```
Comment (je-cook) [inline bluemira/optimisation/_nlopt/conditions.py:64]: Please remove the return types from the docstrings they are in the function definition. You just need the ':' as commented in the original issue

Comment (athoynilimanew) [reply bluemira/optimisation/_nlopt/conditions.py]: done

File: bluemira/display/plotter.py [hunk 2]
```diff
@@ -344,7 +369,15 @@ def _set_label_3d(self):
         self.ax.set_zlabel(offset + Z_LABEL)
 
     def plot_2d(self, obj, ax=None, *, show: bool = True):
-        """2D plotting method"""
+        """
+        2D plotting method
+
+        Returns
+        -------
+        matplotlib.axes.Axes
```
[the remark this answers is not in the document]
Comment (athoynilimanew) [reply bluemira/display/plotter.py]: done

File: bluemira/display/plotter.py [hunk 3]
```diff
@@ -378,7 +411,15 @@ def _make_plot_3d(self):
         """
 
     def plot_3d(self, obj, ax=None, *, show: bool = True):
-        """3D plotting method"""
+        """
+        3D plotting method
+
+        Returns
+        -------
+        matplotlib.axes.Axes
```
[the remark this answers is not in the document]
Comment (athoynilimanew) [reply bluemira/display/plotter.py]: done

**Review by je-cook — APPROVED** [review]

**Pull request merged**